### ISSUE TITLE
[com_fields] Media field Directory option does not follow Media > Options settings for Path

### ISSUE THREAD

**Issue by frogydiak** [joomla/joomla-cms#18514]
### Steps to reproduce the issue
1. Change the default media path (/images) under Content > Media > Options to whatever, I changed it to **assets**. 

2. Create a User Custom Field Media and click on Directory option.

You will see the directory option will still lists all the default files and folders under /images.

### Expected result
The directory option should have respected the path set in Media > Options.

https://monosnap.com/file/biD48ThmkBIUOguRlAc7lJUAF45iiT

### Actual result
Still defaults to images which is the Joomla default.

### System information (as much as possible)
php: Linux cpanel.lakeavechurch.org 2.6.32-673.26.1.lve1.4.18.el6.x86_64 #1 SMP Fri Oct 21 11:58:14 EDT 2016 x86_64
dbversion: 5.5.57-cll
dbcollation: utf8_general_ci
dbconnectioncollation: utf8mb4_general_ci
phpversion: 7.0.22
server: Apache
sapi_name: cgi-fcgi
version: Joomla! 3.8.1 Stable [ Amani ] 4-October-2017 14:00 GMT
platform: Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
useragent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36

**Comment by ghost** [comment]
closed as duplicate Report of #16413.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18514">issues.joomla.org/tracker/joomla-cms/18514</a>.</sub>

**Comment by laoneo** [comment]
This is a sligthly different issue than #16413.

**Comment by ghost** [comment]
reopened as stated above. Thanks for Hint @laoneo<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18514">issues.joomla.org/tracker/joomla-cms/18514</a>.</sub>

**Comment by frogydiak** [comment]
#16413 is about imagelist custom field, I'm pointing on the image and files folder set in the media options.
![joomla media_options](https://user-images.githubusercontent.com/1373815/34505817-8f29065e-efdc-11e7-8807-2149ff4795c4.png)

**Comment by jwaisner** [comment]
@frogydiak Is this still an issue in the latest J3?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18514">issues.joomla.org/tracker/joomla-cms/18514</a>.</sub>

**Comment by frogydiak** [comment]
> @frogydiak Is this still an issue in the latest J3?
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/18514](https://issues.joomla.org/tracker/joomla-cms/18514).

@jwaisner  Yes it is I believe, if not just media field, all custom fields when applicable defaults to /images folder.

**Comment by brianteeman** [comment]
please replace the label with j4 issue

**Comment by CSGoat0** [comment]
I think #18514 isn't a bug and is intentional. But it left us with a question, why we are allowed to change `images` in the options when we shouldn't?
Which one should we do? 
- Disable the field that contains `images`?
- Change the functionality to accept another root?

I think the first option is the most reasonable for now according to [this](https://github.com/joomla/Manual/blob/main/versioned_docs/version-4.4/general-concepts/forms-fields/standard-fields/accessiblemedia.md#:~:text=directory%20(optional)%20(translatable)%20is%20the%20directory%20from%20which%20the%20user%20will%20be%20able%20to%20choose%20a%20file.%20This%20attribute%20should%20be%20relative%20to%20the%20top%20level%20/images/%20folder.%20The%20default%20is%20that%20the%20user%20will%20be%20confined%20to%20the%20top%20level%20/images/%20folder.).

**Comment by Milo-W** [comment]
OK I think I have a possible problem:
strange non-utf or something signs in the file names
something like:

<img width="354" height="41" alt="Image" src="https://github.com/user-attachments/assets/827b2386-5db0-4d48-874d-ec757991224a" />

If I remove it or change it via SSH, the folder comes up. If not, it loads forever.

To consider whether it's a bug - or maybe there should be some info about it or something.

Even normal UTF ółęść in file names aren't a problem, but those are.

I will investigate what happened and why on my own but the thing is - should the panel somehow treat those as I think people tend to do this - name files with their country specific characters (I know it's bad, plus no spaces)...
